# Sort of property field to avoid general error (`pit/column-order`)

âš ï¸ This rule _warns_ in the `base` config.

ğŸ”§ This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->


## Rule Details

å¯¹form-baseä¸­çš„columnså­—æ®µè¿›è¡Œå€¼æ’åºï¼Œé¿å…å‡ºç°å­—æ®µä½ç½®å„ä¸ç›¸åŒçš„é—®é¢˜ã€‚

é»˜è®¤é¡ºåºä¸ºï¼š['äºº', 'å•ä½', 'ç»„ç»‡', 'æ—¶é—´', 'æµç¨‹']

åœ¨"äºº"ä¹‹å‰çš„å­—æ®µï¼Œç»Ÿä¸€æŒ‰é¡ºåºæ”¾åœ¨"äººâ€œä¹‹å‰ï¼Œâ€æµç¨‹â€œä¹‹åçš„å­—æ®µï¼Œç»Ÿä¸€æŒ‰é¡ºåºæ”¾åœ¨â€æµç¨‹ä¹‹åâ€œã€‚

Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```
## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
