# Sort of property field to avoid general error (`pit/column-order`)

⚠️ This rule _warns_ in the `base` config.

🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->


## Rule Details

对form-base中的columns字段进行值排序，避免出现字段位置各不相同的问题。

默认顺序为：['人', '单位', '组织', '时间', '流程']

在"人"之前的字段，统一按顺序放在"人“之前，”流程“之后的字段，统一按顺序放在”流程之后“。

Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```
## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
